cms/envs/common.py

```diff
@@ -994,6 +994,9 @@
 XBLOCK_SETTINGS = {
     "VideoDescriptor": {
         "licensing_enabled": FEATURES.get("LICENSING", False)
+    },
+    'VideoModule': {
+        'YOUTUBE_API_KEY': YOUTUBE_API_KEY
```

Comment:
@e0d  Here is more detail about API key;
As we are using API[v3.0](https://developers.google.com/youtube/v3/getting-started) to fetch the video metadata and that api requires key for authentication. 
To get the key we need an app on Google Developers Console. [Here](https://developers.google.com/youtube/registering_an_application) are the detailed steps to do so. 

I'm not sure how we can automate this stuff. What i'm thinking right now is to add test app for sand-boxes and devstack and using keys from those test projects.